integration/backward_compatibility_test.go

```diff
@@ -16,24 +16,34 @@ import (
 	"github.com/cortexproject/cortex/integration/e2ecortex"
 )
 
+type serviceSetupFn func(name string, s *e2ecortex.CortexService)
+
 var (
 	// If you change the image tag, remember to update it in the preloading done
 	// by CircleCI too (see .circleci/config.yml).
-	previousVersionImages = []string{
-		"quay.io/cortexproject/cortex:v0.6.0",
-		"quay.io/cortexproject/cortex:v0.7.0",
+	previousVersionImages = map[string]serviceSetupFn{
+		"quay.io/cortexproject/cortex:v0.6.0": func(name string, s *e2ecortex.CortexService) {
```

Comment:
What do you think if we change the `NewHTTPReadinessProbe` instead to accept a response status code range, so that we can just configure Cortex services to use the range `200-299` as success?

Reply:
Great idea!